app/server.go

```diff
@@ -35,6 +35,9 @@ func main() {
 
 	directory := filesystem.NewDirectory(os.Getenv("ETHER_CONTENT_DIR"))
 
+	client := &http.Client{}
+	karen := os.Getenv("KAREN_HOST")
```

Comment:
```suggestion
	karen := os.Getenv("KAREN_SERVER")
```
Just to be consistent with the naming scheme in `patches` right now.